kufar/books/Dockerfile

```diff
@@ -2,4 +2,9 @@ FROM golang:latest
 WORKDIR /app
 COPY . .
 RUN go build -o main .
+
+FROM alpine:latest
+RUN apk --no-cache add ca-certificates
+WORKDIR /root/
+COPY --from=0 /app .
```

Comment:
Concept is right. But you copying all `/app` folder with all source codes. You don't need to do this. Just copy your binary and any run-time dependencies (as configs, migrations etc) if you have it.